CODE_OF_CONDUCT.md

```diff
@@ -69,7 +69,7 @@ dispute. If you are unable to resolve the matter for any reason, or if the
 behavior is threatening or harassing, report it. We are dedicated to providing
 an environment where participants feel welcome and safe.
 
-Reports should be directed to snowflake@anyonsys.com.
+Reports should be directed to snowflurry@anyonsys.com.
```

Comment:
We just need to be sure that we have this account set up (and forward mail from `snowflake@anyonsys.com` to `snowflurry@anyonsys.com`).